coderunners/scoring.py

```diff
@@ -0,0 +1,26 @@
+from abc import abstractmethod
+
+from models import Status, TestGroup
+
+
+class AbstractScorer:
+    @abstractmethod
+    def get_score(self, test_results: list[Status], test_scores: list[int]) -> float:
+        raise NotImplementedError()
```

Comment:
If we inherit from ABC and annotate with `@abstractmethod`, we can have `...` here instead of raising NotImplementedError